### PR TITLE
Fiid 95/resumen de inversion

### PR DESCRIPTION
**Pull request by francoFinniu**
- "Resument de inversion para fondo empresarial"  [FIID-95](https://finniu-tech.atlassian.net/browse/FIID-95)
- AnimationNumber is added that takes a number and gives it the effect of going up
- Modal Investment Summary is added, which contains CloseButton(), TitleModal(), IconFond(), InvestmentAmountCardsRow(), TermProfitabilityRow(), SelectedBank(), InvestmentEnds(),
- All widgets include light/dark mode and contain their color change variables within

Screenshot 2024-07-05 180514
Screenshot 2024-07-05 180534

### CODE REVIEW

File: lib/presentation/screens/catalog/widgets/modal_investment_summary.dart [hunk 1]
```diff
@@ -0,0 +1,542 @@
...
+
+class BodyModalInvestment extends ConsumerWidget {
+  const BodyModalInvestment({
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final isDarkMode = ref.watch(settingsNotifierProvider).isDarkMode;
+    const backgroudDark = 0xff1A1A1A;
```
Comment (harolCalzada) [inline lib/presentation/screens/catalog/widgets/modal_investment_summary.dart:33]: spelling

File: lib/presentation/screens/catalog/widgets/modal_investment_summary.dart [hunk 2]
```diff
@@ -0,0 +1,542 @@
...
+    const backgroudLight = 0xffFFFFFF;
```
Comment (harolCalzada) [inline lib/presentation/screens/catalog/widgets/modal_investment_summary.dart:34]: spelling

File: lib/presentation/screens/catalog/widgets/modal_investment_summary.dart [hunk 3]
```diff
@@ -0,0 +1,542 @@
...
+              ),
+            ],
+          ),
+        ),
+      ],
+    );
+  }
+}
+
+class IconFond extends ConsumerWidget {
```
Comment (harolCalzada) [inline lib/presentation/screens/catalog/widgets/modal_investment_summary.dart:393]: Icon Fund

**Review by harolCalzada — CHANGES_REQUESTED** [review]

File: lib/presentation/screens/catalog/widgets/modal_investment_summary.dart (hunk 1 above)
Comment (francoFinniu) [reply lib/presentation/screens/catalog/widgets/modal_investment_summary.dart]: fix error

File: lib/presentation/screens/catalog/widgets/modal_investment_summary.dart (hunk 2 above)
Comment (francoFinniu) [reply lib/presentation/screens/catalog/widgets/modal_investment_summary.dart]: fix error

File: lib/presentation/screens/catalog/widgets/modal_investment_summary.dart (hunk 3 above)
Comment (francoFinniu) [reply lib/presentation/screens/catalog/widgets/modal_investment_summary.dart]: fix error

**Review by harolCalzada — APPROVED** [review]

**Pull request merged**